src/lib/app-helper.js

```diff
@@ -567,6 +567,29 @@ function getObjectValue (obj, key) {
   return keys.filter(o => o.trim()).reduce((o, i) => o && getObjectProp(o, i), obj)
 }
 
+/**
+ * Function to run `aio config get ims.contexts.cli.access_token.token` command and retrieve the token
+ * This function is used to check if the access token exists in the CLI context or not
+ * @returns {Promise<boolean>} Resolves to the access token if it exists, else false
+ */
+const checkifAccessTokenExists = async () => {
+  try {
+    const { stdout, stderr } = await execa('aio', ['config', 'get', 'ims.contexts.cli.access_token.token'])
```

Comment:
do not go through all this extra ceremony of calling the cli for another command just to read config.  The config is likely already loaded and in mem, even this file has other references to reading values from config without extra trips + exec.
Look at ln:21 of this file.